Add option to include static generated pages (#185)
nuxt/now-builder: README.md

}
```

> ### `useGenerate`

Reviewer: I think we can make this more intuitive:

```suggestion
### `generateStaticRoutes`
```

If you agree, would you make this change throughout?

Author: Agreed!

- Type: `Boolean`
- Default: `false`

> To pre-render routes during the build using `nuxt generate`, set this to true. Routes that are not generated will fallback to the server lambda. Use [Nuxt.js' exclude option](https://nuxtjs.org/api/configuration-generate#exclude) to disable generation of certain routes.

Reviewer: ```suggestion
To pre-render routes during the build using `nuxt generate`, set this to true. Routes that are not generated will fallback to the server lambda. You will need to [specify the routes to be generated](https://nuxtjs.org/api/configuration-generate/#routes) in your `nuxt.config`.
```
